book.rb

```diff
@@ -0,0 +1,15 @@
+require './person_class'
+require './rental'
```

Comment:
- [OPTIONAL] I will highly recommend you to use `require_relative` here as we only use relative for external files as per Ruby convention.